models/unite-legale.ts

```diff
@@ -206,13 +206,17 @@ const getUniteLegale = async (
         // no pagination as this function is called when sirene etalab already failed
         return await fetchUniteLegaleFromInseeFallback(siren, page);
       } catch (lastFallback: any) {
+        if (lastFallback instanceof HttpNotFound) {
```

Comment:
I expect the fallback logic to be in a client rather than in a model. but i do not now the codebase so it may be rightfull to do it this way.

Reply:
Les fallback amènent a appeler des clients d'API différents et comportent de nombreux concepts métiers, c'est pour ca que je les avais placé dans le model